runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -215,13 +218,23 @@ void rebuild()
 			priceTypeString = priceType == LootTrackerPriceType.HIGH_ALCHEMY ? "HA: " : "GE: ";
 		}
 
-		priceLabel.setText(priceTypeString + QuantityFormatter.quantityToStackSize(totalPrice) + " gp");
-		priceLabel.setToolTipText(QuantityFormatter.formatNumber(totalPrice) + " gp");
+		String averageString = kills > 1 ? QuantityFormatter.formatNumber(totalPrice / kills) + " gp" : "0";
+		if (showAverage)
+		{
+			priceLabel.setText(averageString);
+			priceLabel.setToolTipText(priceTypeString + QuantityFormatter.formatNumber(totalPrice) + " gp (total)");
+		}
+		else
+		{
+			priceLabel.setText(priceTypeString + QuantityFormatter.quantityToStackSize(totalPrice) + " gp");
+			priceLabel.setToolTipText("<html>" + QuantityFormatter.formatNumber(totalPrice) + " gp<br>" + averageString + " (average)</html>");
+		}
 
 		final long kills = getTotalKills();
 		if (kills > 1)
 		{
 			subTitleLabel.setText("x " + kills);
+			subTitleLabel.setToolTipText(averageString + " (average)");
```

Comment:
Why is this tooltip added here? Given that this is price information, it should only be handled with the price data on the right, which is already covered in the code above.

Reply:
This was my original way of doing it.  I hovered over the kill count, hoping it would show me the average.  Maybe others have the same approach.

If you're okay for it to be left it, I think it's a good location to always be able to see the average, without clicking the button.  I can remove it if you'd prefer though.

Reply:
I don't think it belongs here, but I'll let others comment with their feelings on this.